src/main/java/com/softserveinc/ita/pageobjects/HomePage.java

```diff
@@ -1,5 +1,18 @@
 package com.softserveinc.ita.pageobjects;
 
+import static com.codeborne.selenide.Selenide.$x;
+import static java.lang.String.format;
+
 public class HomePage extends BasePage<HomePage> {
 
+    public String getLastProductOfSection(String sectionName) {
```

Comment:
probably create a separate method for getting the title of the last product of section and method which would open specific product in section by title and return product page